build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 RELEASE=r6p0
-
+PARALLEL=-j$(nproc)
```

Comment:
Sorry if this wasn't clear enough in the previous review, but it would be great if that variable was also renamed to JOBS.

(so sed -i 's/PARALLEL/JOBS/' on that whole patch).

Thanks!

Reply:
Sure, good idea, it's more clear.